e2e/config.js

```diff
@@ -31,6 +31,10 @@ module.exports = {
     email: 'hmcts-admin@example.com',
     password: process.env.CA_USER_PASSWORD || defaultPassword,
   },
+  hmctsSuperUser: {
+    email: 'hmcts-superuser@example.com',
+    password: process.env.CA_USER_PASSWORD || defaultPassword,
+  },
```

Comment:
are we happy for this to have the court admin user password? or do we need another one like SUPER_USER_PASSWORD?

Reply:
Probably should be, do you know if we set those env variables anywhere?

Reply:
I didn't find where we set ! Does anyone know? or its probably not set and using defaultPassword everywhere 😄

Reply:
Looks like the only place we set the vars are for the smoke test and that is for the `SMOKE_TEST_LA_USER_USERNAME` and `SMOKE_TEST_LA_USER_PASSWORD`. I think keeping it different is sensible however as if we do decide to set the variable it may be different from the admin one

Reply:
I can see we set SYSTEM_UPDATE_USER_USERNAME in jenkins cnp which we read from key vault. for the rest seems to me like not set and use defaultPassword but not 100% sure (looks that way though)

Reply:
don't know where we are going to set password but i have added new env variable - SUPER_USER_PASSWORD

Reply:
Think it's just if we want to set it in future the env variable is there :)